### DISCUSSION TITLE
associate file with another file type, for formatting

### DISCUSSION THREAD

**Discussion by aum7** [LunarVim/LunarVim#3820] [Q&A] [answered]
olo
i need to associate .ld file type (luadoc / ldoc)
so (when typing & when saved) it will be formatted as .lua file
i checked config.lua file, i think i need language server
lvim.lsp... settings
maybe
formatters...
or  maybe
linters.setup...

stucked here.
what i want, is when i type | save file.ld, to be formatted as file.lua file
ie at the end of -- comment line
when i press enter, new line would already be -- commented
-- and ready to rock
and when i save the file.ld, it will be formatted as any.lua file
  
thank you for your help

godspeed
aum
s love nia

**Comment by LostInTheLogs** [comment, marked as answer]
`vim.filetype.add({extension = { ld = "lua" }})`

**Reply by aum7** [reply]
beautiful
works like a charm
thank you
bow
godspeed
